src/swagger.js

```diff
@@ -74,9 +74,9 @@ const OrganizationAndUser = Joi.object({
 
 const List = (data) => {
   return Joi.object({
-    page: Joi.number().integer().positive().description('Page number, starts from 1'),
```

Comment:
@mihaidma We need to specify in the doc that pagination will return an error if you pass `page` or `limit` equal to zero.